### PR TITLE
Remaking the 404 NotFound Page

### LINKED ISSUE #57: Remaking the NotFound (Error) page
### Description

Remake the NotFound page according to the new wireframe provided using bootstrap and custom CSS. This should be made responsive. I'll be sending the assets for this page once someone is assigned.

### Screenshots/testimonials

Desktop - 2

### Concerning Files/Folders

https://github.com/houseofgeeks/hg/tree/front-end/client/src/components/Error

### PR DESCRIPTION
**Pull request by Abhayraj1707**
fixes #57
<Do not delete this template>
Description: <provide a description> Remaking of error page.
As per wireframe I have tried my best .here i added one more thing that astronaut is moving like in space.
this page is responsive for most of the devices, because I don't have a wireframe for small devices, i have  make it like this.
I added one more thing  i.e., stars in the background that's make it beautiful. Also a back button is given and after clicking on this we reach to landing page .Any suggestion is welcome .

#### Checklist
- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes which were demanded in the linked issue.
- [x] I've tested my code on a Chromium based browser.
- [x] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial
<provide a relevant screenshot or any testimonial for the solved issue if applicable>

error_page
error_page1

### CODE REVIEW

File: client/src/components/Error/Error.css
Comment (EmperorYP7) [inline client/src/components/Error/Error.css]: oops. Looks like every `ul` in the website is picking it up. You should go for `.error-component ul`

File: client/src/components/Error/Error.css
Comment (EmperorYP7) [inline client/src/components/Error/Error.css]: So, every property stated in `Error.css` must have `.error-component` before it to target only `Error.js` component.

File: client/src/components/Error/Error.js
Comment (EmperorYP7) [inline client/src/components/Error/Error.js]: add `className="error-component` here.

**Review by EmperorYP7 — CHANGES_REQUESTED** [review]
Also, the mobile view isn't looking right. Use `vh` (view height) and `vw` (view width) for `font-size, margin, height` etc.. This might help. The mobile view for this page has not been decided yet in the wireframe. So, try resizing the SVGs until it looks good enough.

File: client/src/components/Error/Error.js
Comment (tusharjain0022) [inline client/src/components/Error/Error.js]: Suggestion : In place of  `GO BACK`  here , it should be `Home` or `Back to Home` . I think it would be more accurate here according to its purpose/link.

Comment (Abhayraj1707) [reply client/src/components/Error/Error.js]: okk I will resolve this in my next commit .thanks  for suggestion!!

**Comment by ankiiitraj** [comment]
Updates on this! @Abhayraj1707

**Comment by Abhayraj1707** [comment]
> Updates on this! @Abhayraj1707

sir i m working on this but its seems hard for me to making it more responsive as css part is little bit diffcult. Today i will make a commit and after that if @tusharjain0022  wants to make it better then he will also make commits .

**Comment by EmperorYP7** [comment]
@tusharjain0022 have a look here, try to fix the styling and conflicts..

**Comment by tusharjain0022** [comment]
@Abhayraj1707 I've made a commit to your fork for resolving the conflicts with the branch and solving the issue of conflicting css classes.

**Comment by EmperorYP7** [comment]
@ankiiitraj @tusharjain0022 you can review this again now.

**Comment by hariom1625** [comment]
Suggestion: Some styling can be added to the Home button present on the top left corner it looks quite plain.
The link isn't active on the Home button, you can use a styled button or an arrow instead of Navlink :-)

**Comment by EmperorYP7** [comment]
Now its somewhat ready. I think we should get this done right away. This has been stalled for quite some time.

**Review by tusharjain0022 — APPROVED** [review]
LGTM! 🙌

**Review by EmperorYP7 — APPROVED** [review]

**Comment by EmperorYP7** [comment]
@ankiiitraj waiting for your review..

**Comment by ankiiitraj** [comment]
Just one tweak there's on overflow in the horizontal direction in small screens!

**Review by ankiiitraj — APPROVED** [review]

**Comment by EmperorYP7** [comment]
Tried to fix this. Seems like something's not happy. We can let go of this for now.

**Comment by hariom1625** [comment]
> Tried to fix this. Seems like something's not happy. We can let go of this for now.

Pushing some changes to @Abhayraj1707 fork, understood what's causing the overflow.

**Review by hariom1625 — APPROVED** [review]

**Comment by EmperorYP7** [comment]
still a little horizontal movement present in the page.

**Comment by EmperorYP7** [comment]
Merging this since every maintainer has approved it. Creating a new issue for the horizontal movement.

**Pull request merged**
